transformable-image.js

```diff
@@ -47,15 +47,19 @@ export default class TransformableImage extends React.Component {
       inputRange: [-100, 100],
       outputRange: ['-100rad', '100rad'],
     });
+    this._rotateY = this._rotate.interpolate({
```

Comment:
Wow, nice! Is this working as a mirror, because for the rotation the input and the output ranges are shifted by 180 degrees?

Reply:
That's it!